ui-components/src/locales/general.json

```diff
@@ -439,13 +441,21 @@
+  "application.start.whatToExpect.fcfs.steps": "1. First we'll ask about you and the people you plan to live with.\n2. Then, we'll ask about your income.\n3. Finally, we'll see if you qualify for any affordable housing lottery preferences, if applicable.",
+  "application.start.whatToExpect.fcfs.finePrint": "* Applicants are applying to currently vacant apartments on a first come, first serve basis.\n* Please be aware that each household member can only appear on one application for each listing.\n* Applicants will be contacted by the property manager on a first come, first serve basis, until vacancies are filled.\n* All of the information that you have provided will be verified and your eligibility confirmed.\n* Your application may be removed if you have made any fraudulent statements.\n* For properties with housing preferences, if we cannot verify a housing preference that you have claimed, you will not receive the preference but will not be otherwise penalized.",
+
+  "application.start.whatToExpect.lottery.steps": "1. First we'll ask about you and the people you plan to live with.\n2. Then, we'll ask about your income.\n3. Finally, we'll see if you qualify for any affordable housing lottery preferences, if applicable.",
+  "application.start.whatToExpect.lottery.finePrint": "* Applicants are applying to enter a lottery for currently vacant apartments.\n*  Please be aware that each household member can only appear on one application for each listing.\n*  Applicants will be contacted by the property agent in lottery rank order until vacancies are filled.\n*  All of the information that you have provided will be verified and your eligibility confirmed.\n*  Your application may be removed if you have made any fraudulent statements.\n*  For properties with housing preferences, if we cannot verify a housing preference that you have claimed, you will not receive the preference but will not be otherwise penalized.",
+
+  "application.start.whatToExpect.waitlist.steps": "1. First we'll ask about you and the people you plan to live with.\n2. Then, we'll ask about your income.\n3. Finally, we'll see if you qualify for any affordable housing preferences, if applicable.",
+  "application.start.whatToExpect.waitlist.finePrint": "* Applicants are applying for an open waitlist and not a currently available apartment.\n* Please be aware that each household member can only appear on one application for each listing.\n* When vacancies become available, eligible applicants will contacted by the property manager on a first come, first serve basis.\n* All of the information that you have provided will be verified and your eligibility confirmed.\n* Your application may be removed if you have made any fraudulent statements.\n* For properties with housing preferences, if we cannot verify a housing preference that you have claimed, you will not receive the preference but will not be otherwise penalized.",
```

Comment:
There is a typo in the copy here - need to update `will contacted` to `will be contacted`